### DISCUSSION TITLE
Trying to add in DLLs for QT6 to work with QMultimedia

### DISCUSSION THREAD

**Discussion by djotaku** [pyinstaller/pyinstaller#8421] [Help]
My repo is at https://github.com/djotaku/ELDonationTracker/tree/qt6

The errors I'm getting are:
17450 INFO: Extra DLL search directories (AddDllDirectory): ['C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x64\\Lib\\site-packages\\PyQt6\\Qt6\\bin']
17450 INFO: Extra DLL search directories (PATH): ['C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x[64](https://github.com/djotaku/ELDonationTracker/actions/runs/8760964609/job/24046774231?pr=213#step:5:65)\\Lib\\site-packages\\PyQt6\\Qt6\\bin']
18638 WARNING: Library not found: could not resolve 'avcodec-60.dll', dependency of 'C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x64\\Lib\\site-packages\\PyQt6\\Qt6\\plugins\\multimedia\\ffmpegmediaplugin.dll'.
18638 WARNING: Library not found: could not resolve 'avformat-60.dll', dependency of 'C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x64\\Lib\\site-packages\\PyQt6\\Qt6\\plugins\\multimedia\\ffmpegmediaplugin.dll'.
18638 WARNING: Library not found: could not resolve 'avutil-58.dll', dependency of 'C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x64\\Lib\\site-packages\\PyQt6\\Qt6\\plugins\\multimedia\\ffmpegmediaplugin.dll'.
18638 WARNING: Library not found: could not resolve 'swscale-7.dll', dependency of 'C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x64\\Lib\\site-packages\\PyQt6\\Qt6\\plugins\\multimedia\\ffmpegmediaplugin.dll'.
18638 WARNING: Library not found: could not resolve 'swresample-4.dll', dependency of 'C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x64\\Lib\\site-packages\\PyQt6\\Qt6\\plugins\\multimedia\\ffmpegmediaplugin.dll'.

I have tried adding the ffmpeg directory in my repo where I have those DLLs to the --add-data and to the -p and it still doesn't find them. What do I need to do to get this to find the DLLs?

Thanks!

**Comment by rokm** [comment]
Add the location with those DLLs to `PATH` environment variable before you run PyInstaller.

**Comment by djotaku** [comment]
Just in case someone finds this in the future. I tried doing that literally a dozen different ways and it didn't work. In the end, I just manually copied those files into the directory that pyinstaller was looking for them in C:\\hostedtoolcache\\windows\\Python\\3.11.9\\x64\\Lib\\site-packages\\PyQt6\\Qt6\\bin and that worked.
